SPECS/kata-containers-cc/kata-containers-cc.signatures.json

```diff
@@ -1,6 +1,6 @@
 {
   "Signatures": {
-    "kata-containers-3.2.0.azl4-cargo.tar.gz": "2a242deedddbd01b50b56d9f6d02ffd3f40cb2e91221fda4f4b4791d98404f96",
```

Comment:
please remove these changes for a pull request. please prepare or contribute to a branch that can be used as the base branch for the Kata CI pipeline which ingests the new release sources. The PR title should then be 'update kata-containers package to azl3.2.0.5 or similar'